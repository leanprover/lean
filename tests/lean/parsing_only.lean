--

constant f : nat → nat
constant g : nat → nat
notation a `+++` := f a
notation [parsing_only] a `***` := g a
#check 10 +++
#check 10 ***
#check Type.{8}  -- Type₊ should not be used
