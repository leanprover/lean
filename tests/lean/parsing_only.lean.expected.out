10+++ : ℕ
g 10 : ℕ
Type 8 : Type 9
